lambdas/es/indexer/index.py

```diff
@@ -168,8 +168,17 @@ def handler(event, _):
     dispatch post_to_es
     """
     try:
-        for msg in event['Records']:
-            for record in json.loads(json.loads(msg['body'])['Message'])['Records']:
+        # existence of event['Records'] is guaranteed by
+        # https://docs.aws.amazon.com/AmazonS3/latest/dev/notification-content-structure.html
+        # see above for details on event structure, s3:TestEvent
+        for outer in event['Records']:
+            # these should all be SNS messages (indexer consumes from SNS topic, by design)
+            raw_message = json.loads(outer['body'])['Message']
+            message = json.loads(raw_message)
+            if 'Records' not in message:
+                # consume event (we don't want to index it)
```

Comment:
Are we sure to be the only consumer of the events in this Queue? If not, it might be worth writing a special case for TestEvent and simply ignoring (not consuming) other events.

Reply:
@stevededalus is there a way to ignore events without consuming them? from what i read below, consumption == lambda task did not throw.